rdb/src/main/java/com/instana/dc/rdb/impl/InformixDc.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ * (c) Copyright Instana Inc.
+ */
+package com.instana.dc.rdb.impl;
+
+import com.instana.dc.CalculationMode;
+import com.instana.dc.rdb.AbstractDbDc;
+import com.instana.dc.rdb.DbDcUtil;
+import com.instana.dc.rdb.util.Constants;
+import com.instana.dc.rdb.util.InformixUtil;
```

Comment:
It is not required to for each implementation to have a source file in "impl" package and another in "util" package. They can use any way. I prefer to have a special package for each implementation. For example: "impl/informix" package for informix, "impl/dameng" package for DamengDB. In each package, the implementation can be very different. Not have to use the same JDBC style.

Reply:
The package structure has been modified